app/models/concerns/orchestration.rb

```diff
@@ -82,14 +90,22 @@ def record_conflicts
     @record_conflicts ||= []
   end
 
+  def skip_orchestration!
```

Comment:
This looks very similar to the mechanism already added in 5351c86, so I don't think we should have both.  The only difference seems to be in that `importing_facts` prevents queueing, and this prevents execution of queues.

This is more generic and better named, so I think you should replace `importing_facts` with `skip_orchestration`.